src/_actions/TradeActions.js

```diff
@@ -82,81 +82,85 @@ export const updateTradeError = (index, errorID, error) => ({
     error,
 });
 
-export const updatePriceProposalSubscription = (tradeID, trade) => {
-    const thunk = (dispatch, getState) => {
-        dispatch(updateTradeUIState(tradeID, 'disabled', true));
-        if (!getState().tradesParams.get(tradeID)) {
-            return;
-        }
-        const tradeParam = trade || getState().tradesParams.get(tradeID).toJS();
-        const { proposal } = getState().tradesProposalInfo.get(tradeID).toJS();
-        const currency = getState().account.get('currency');
-        const {
-            amount,
-            basis,
-            type,
-            dateStart,
-            duration,
-            durationUnit,
-            symbol,
-            barrier,
-            barrier2,
-            amountPerPoint,
-            stopType,
-            stopProfit,
-            stopLoss,
-            barrierType,
-        } = tradeParam;
-
-        if (!(amount && basis && type && symbol)) {
-            return;
-        }
+const subscriptionQueue = [
```

Comment:
Good that you are taking this into consideration.
Could be too much of an edge case, and not happen really in practice (or it could).